docs/Get started/why-fleet.md

```diff
@@ -1,7 +1,11 @@
 # Why Fleet
 
-## What's it for?
-Organizations like Fastly and Gusto use Fleet for vulnerability reporting, detection engineering, device management (MDM), device health monitoring, posture-based access control, managing unused software licenses, and more.
+In an industry that views security and IT as a product rather than a process, it’s little wonder that most IT and security teams are suffocating in the cognitive overload of bolt-on solutions, misconfigured tools, and disparate workflows. 
```

Comment:
```suggestion
In an industry that views security and IT as a shopping spree rather than a process, it’s little wonder that IT and security teams get stuck with bolt-on solutions, misconfigured tools, and disparate workflows. 
```